Sources/Sentry/include/SentryInstallation.h

```diff
@@ -1,12 +1,13 @@
 #import <Foundation/Foundation.h>
 
 #import "SentryDefines.h"
+#import "SentryOptions.h"
 
 NS_ASSUME_NONNULL_BEGIN
 
 @interface SentryInstallation : NSObject
 
-+ (NSString *)id;
++ (NSString *)idWithOptions:(SentryOptions *)options;
```

Comment:
In case we want the change suggested in the thread at https://github.com/getsentry/sentry-cocoa/pull/2711/files#r1114263689
```suggestion
/** To use the default cache directory, pass @c nil . */
+ (NSString *)idWithCacheDirectory:(nullable NSString *)cacheDirectory
```